syntax = "proto2";
option cc_generic_services = true;

package brpc;

message EchoReq
{
	required string message = 1;
};

message EchoRes
{
	required string message = 1;
};

service EchoService
{
	rpc Echo(EchoReq) returns (EchoRes);
};
